render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlFunctionSerializer.kt

```diff
@@ -0,0 +1,29 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl.JpqlFunction
+import com.linecorp.kotlinjdsl.render.RenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializer
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import kotlin.reflect.KClass
+
+class JpqlFunctionSerializer : JpqlSerializer<JpqlFunction<*>> {
+    override fun handledType(): KClass<JpqlFunction<*>> {
+        return JpqlFunction::class
+    }
+
+    override fun serialize(part: JpqlFunction<*>, writer: JpqlWriter, context: RenderContext) {
+        val delegate = context.getValue(JpqlRenderSerializer)
+
+        writer.write("FUNCTION")
+        writer.write("(")
+
+        writer.write(part.name)
+
+        writer.writeEach(part.args, separator = ", ") {
+            delegate.serialize(it, writer, context)
+        }
```

Comment:
I think there should be a comma between the name of the function and the parameters of the function. 

```kotlin
writer.write(part.name)

writer.write(", ")
writer.write(" ")

writer.writeEach(part.args, separator = ", ") {
    delegate.serialize(it, writer, context)
}
```

Reply:
oh I missed it. I reflect your review in the commit below.

[3fd2f29](https://github.com/line/kotlin-jdsl/pull/421/commits/3fd2f29a59f6674770f08cde8849cc6a62fd9434)